test/clj/game_test/cards/operations.clj

```diff
@@ -1890,26 +1890,33 @@
     (is (= 1 (:click (get-corp))) "Spent 2 clicks")))
 
 (deftest reverse-infection
-  ;; Reverse Infection - purge and trash 1 card from stack for every 3 counters purged - or gain 2 credits
+  ;; Reverse Infection
+  (testing "Gain 2 credits"
+    (do-game
+      (new-game {:corp {:deck ["Reverse Infection"]}})
+      (play-from-hand state :corp "Reverse Infection")
+      (click-prompt state :corp "Gain 2 [Credits]")
+      (is (= 7 (:credit (get-corp))) "Corp gained 2 credits")))
+  (testing "Purge virus counters, counts both Runner and Corp side (Sandstone)")
   (do-game
-    (new-game {:corp {:deck [(qty "Reverse Infection" 2)]}
-               :runner {:deck ["Virus Breeding Ground" "Datasucker" (qty "Sure Gamble" 3)]}})
+    (new-game {:corp {:deck ["Reverse Infection" "Sandstone"]}
+               :runner {:deck ["Virus Breeding Ground" "Datasucker" (qty "Sure Gamble" 4)]}})
     (starting-hand state :runner ["Virus Breeding Ground" "Datasucker"])
```

Comment:
This can be written in the `new-game` hash: `:runner {:deck [...] :hand ["Virus Breeding Ground" "Datasucker"]}`

Reply:
Sweet, learn something new all the time. I wonder if we should just go through and change all cards that use `(starting-hand [...])` to this better way?

Reply:
I've thought about it! Just a tedious piece of work for relatively little gain, lol.